models/table_definitions.py

```
@@ -0,0 +1,798 @@
+
+Usage::
+
+    from models.table_definitions import TableDef, ColumnDef, get_table_definitions
+
+    tables = get_table_definitions()
+    sql = tables["warnings"].to_sql()
+"""
+
+from __future__ import annotations
+
+from pydantic import BaseModel, Field
```

Flake8 (7.3.0)

[error] 'pydantic.Field' imported but unused

(F401)